katello/katello-restore

```diff
@@ -1,24 +1,30 @@
 #!/usr/bin/env ruby
 
+require "socket"
 require 'optparse'
 require 'fileutils'
 require 'date'
 
-STANDARD_BACKUP_FILES = ['config_files.tar.gz', 'pulp_data.tar']
+STANDARD_BACKUP_FILES = ['config_files.tar.gz', 'pulp_data.tar*']
```

Comment:
Nope, that's correct. I thought you were adding a wildcard character, I understand now.